docs/source/configuration/overview.mdx

```diff
@@ -598,9 +598,13 @@ supergraph:
 
 </MinVersion>
 
-<ExperimentalFeature />
+<Note>
+
+Beginning with [v1.49.0](https://github.com/apollographql/router/releases/tag/v1.49.0), the router supported enhanced operation signature normalization [experimentally](/resources/product-launch-stages/#experimental-features).
+In [v.1.53.0](https://github.com/apollographql/router/releases/tag/v1.53.0), this configuration became [generally available](/resources/product-launch-stages/#general-availability).
```

Comment:
The config rename will happen in v1.54.0, so we should use that as the version it became GA.